src/mono/mono/mini/mini-exceptions.c

```diff
@@ -1830,6 +1830,7 @@ ves_icall_get_frame_info (gint32 skip, MonoBoolean need_file_info,
 
 	g_assert (skip >= 0);
 
+#ifndef TARGET_WASM
 	if (mono_llvm_only) {
```

Comment:
This should only affect aot, is this code path really hit in the interpreter case.

Reply:
https://github.com/dotnet/runtime/blob/78bc702926e32baf6573e6d2303d810265435877/src/mono/mono/mini/driver.c#L2988

Reply:
we should move the discussion to https://github.com/mono/mono/pull/20156 though